tooling/src/hypothesistooling/__main__.py

```diff
@@ -435,9 +435,9 @@ def run_tox(task, version, *args):
     "3.9": "3.9.19",
     "3.10": "3.10.14",
     "3.11": "3.11.9",
-    "3.12": "3.12.3",
+    "3.12": "3.12.4",
     "3.13": "3.13.0b2",
```

Comment:
It looks like we may need

```suggestion
    "3.13": "3.13.0b3",
```